hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,9 @@
+RELEASE_TYPE: patch
+
+This patch refactors the strategy classes hypothesis uses internally to implement the
+`floats() <https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.floats>`__
+strategy.
+It should not produce in any behavioral changes.
+This refactoring supports
+`a larger refactoring <https://github.com/HypothesisWorks/hypothesis/issues/3086>`__
+to facilitate integrations with symbolic execution tools and fuzzers.
```

Comment:
```suggestion
This patch by Phillip Schanely refactors some of our internals to support future integrations
with symbolic execution tools and fuzzers (:issue:`3086`).  There is no user-visible change (yet!).
```

I tend to be very, very terse about internals in the changelog - the last thing I want is for users to start thinking that "if it's documented, it's public API", and the issue link is available for the interested.